distribution/lib/Standard/Base/0.0.0-dev/src/Random.enso

```diff
@@ -216,18 +217,22 @@ type Random
          duplicates. If true, values from the `Vector` can be returned multiple
          times and duplicates are possible.
 
-       If `with_replacement` is false, and `count` is equal to or greater than
-       the size of the collection, a random permutation of the collection is
-       returned.
+       ! If `with_replacement` is false, and `count` is equal to the size of the
+         collection, a random permutation of the collection is returned.
+
+       ! If `with_replacement` is false, and `count` is greater than the size of
+         the collection, an `Illegal_Argument` error is thrown. (If `count` is
+         greater than the size of the collection, `items` will run out of values
+         to choose from, and will not be able to return enough values.)
```

Comment:
As noted before, I think the `!` sections need a header - at least IIRC 99% of the ones I'm aware of do have one.

Maybe we can merge this to a single section `? Selection without replacement`? (also nitpick that IMHO `?` feels better here)

Reply:
Done